Hypercube/src/app.php

```diff
@@ -13,10 +15,18 @@
 $app->register(new ServiceControllerServiceProvider());
 $app['hypercube.controller'] = function () use ($config) {
     return new HypercubeController(
-        new TesseractService($config['executable'])
+        new CmdExecuteService(),
```

Comment:
If we do end up wanting to use a logger, we can use the silex monolog implementation like we are doing in Houdini:
https://github.com/Islandora-CLAW/Crayfish/pull/20/files#diff-63757cc0ace6867bee89a08c62fd76f2R17